src/journal/journald-stream.c

```diff
@@ -282,6 +300,18 @@ static int stdout_stream_log(StdoutStream *s, const char *p) {
                         IOVEC_SET_STRING(iovec[n++], syslog_identifier);
         }
 
+        if (line_break != LINE_BREAK_NEWLINE) {
+                const char *c;
+
+                /* If this log message was generated due to an uncommon line break then mention this in the log
+                 * entry */
+
+                c =     line_break == LINE_BREAK_NUL ?         "_LINE_BREAK=nul" :
+                        line_break == LINE_BREAK_MAX_REACHED ? "_LINE_BREAK=max-reached" :
```

Comment:
I'm not convinced about "max-reached", it doesn't say the maximum of what was reached. Wouldn't "max-line" or "line-max" or "line-limit" or "max-length" be better?